simapp/simd/cmd/root.go

```diff
@@ -45,7 +45,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		WithLegacyAmino(encodingConfig.Amino).
 		WithInput(os.Stdin).
 		WithAccountRetriever(types.AccountRetriever{}).
-		WithBroadcastMode(flags.BroadcastBlock).
+		WithBroadcastMode(flags.BroadcastSync).
```

Comment:
@amaurym why was this line even added? It should be taken care of by `readTxCommandFlags`.

Reply:
Why we need Broadcast Sync here?

Reply:
Yeah agrred, we should delte this line

```suggestion
```